.github/workflows/deploy.yml

```diff
@@ -4,12 +4,11 @@ name: Deploy
 
 on:
   schedule:
-    - cron: "*/30 * * * *"
+    - cron: '*/30 * * * *'
 
 jobs:
   deploy:
     name: Scheduled Deployment
-    if: github.repository == 'kawalcovid19/wargabantuwarga.com'
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!